RELEASE_NOTES.md

```diff
@@ -7,9 +7,26 @@ in the release. For a detailed list of changed components, refer to
 This document contains release notes for the current release and all patches.
 For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
-## 12.6.0
+## 12.7.0
 
 ### Chef Server
-  * Fixed false failures for postgres preflight check.
-  * Fixed bug where public\_key\_read\_access group was not properly being respected for access on org-scoped user and client public key reads.
-  * Update licensing handling to make licensing information easier to find (see [here](https://www.chef.io/blog/2016/04/26/changes-to-how-chef-products-handle-licenses/)).
+  * Add support for service credential rotation via [Veil](https://github.com/chef/chef-server/blob/3ff412b5a2e6ad54cfa79bca6865e1bbca28fe5e/omnibus/files/veil/README.md), and for requiring it via the added `require-credential-rotation` command of `chef-server-ctl`.
+  * Fixed existing process detection in oc_id which caused high CPU utilization
+  * Fixed bug where deleting a user would not delete pending invites.
+  * Allow filtering users by their external_authentication_uid, allowing for SAML-authentication in Chef-Manage.
+
+### Security Updates
+
+* The update of OpenSSL 1.0.1t addresses the following:
+  * CVE-2016-2105
```

Comment:
Ok; I've opted for linking the openssl pages, since they've got the easiest-to-consume summaries (IMHO) and link to Mitre, too. What do you think?